source/src/protocols/cyclic_peptide_predict/SimpleCycpepPredictApplication.cc

```diff
@@ -803,6 +813,25 @@ SimpleCycpepPredictApplication::initialize_from_options(
 		}
 	}
 
+	//CWT store lan and melan positions
+	if ( option[basic::options::OptionKeys::cyclic_peptide::lanthionine_positions].user() ) {
+		core::Size const nlanres(option[basic::options::OptionKeys::cyclic_peptide::lanthionine_positions]().size());
+		runtime_assert_string_msg( nlanres > 0, "Error in simple_cycpep_predict application: The \"-cyclic_peptide:lanthionine_positions\" commandline option must be followed by a list of residues to form lanthionine rings." );
+		//runtime_assert_string_msg( nlanres % 2 == 0, "Error in simple_cycpep_predict application: The \"-cyclic_peptide:lanthionine_positions\" commandline option must be followed by a list of residues, where the number of residues in the list is a multiple of two.  Pairs of residues must form lanthionine rings." );
+		//Only allowing two specified residues at the moment
+		runtime_assert_string_msg( nlanres == 2, "Error in simple_cycpep_predict application: The \"-cyclic_peptide:lanthionine_positions\" commandline option must be followed by a list of two residues. Pairs of residues must form lanthionine rings." );
+		core::Size count(0);
+		lanthionine_positions_.resize(nlanres / 2);
+		for ( core::Size i(1), imax(nlanres / 2); i<=imax; ++i ) {
+			utility::vector1 <core::Size> innervect(2);
```

Comment:
vector1 works, but if you know you'll always have just two elements, std::pair is likely to be a better choice. (But no need to re-write things just because.)